dist/@types/models/core/item.d.ts

```diff
@@ -150,6 +150,7 @@ export declare class ItemMutator {
     setDeleted(): void;
     set lastSyncBegan(began: Date);
     set errorDecrypting(errorDecrypting: boolean);
+    set errorDecryptingValueChanged(errorDecryptingValueChanged: boolean);
```

Comment:
Is there no other way to test this than to make a test-specific setter? When else would you use this?

Reply:
Yeah you're right, there should be another way to do this without adding a new otherwise unused function.